### PR TITLE
Update manage-domain-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7394

File: microsoft-365/admin/setup/manage-domain-users.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ To synchronize your users, groups, and contacts from the local Active Directory
 
  1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339), select **Setup** in the left nav.
 
- 2. Under **Sign-in and security**, choose **View**  under **Sync users from your org's directory**.
+ 2. Under **Sign-in and security**, Select **Add or sync users to your Microsoft account**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/manage-domain-users.md]:
```suggestion
 2. Under **Sign-in and security**, select **Add or sync users to your Microsoft account**.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR, Thanks

**Pull request merged**